src/node/connection_types.h

```diff
@@ -74,6 +74,12 @@ enum class ConnectionType {
      * AddrMan is empty.
      */
     ADDR_FETCH,
+
+    /**
+     * Full-rbf connections are long-lived connections used to relay transactions
+     * with peers accepting replacement without requiring replaceability signaling.
+     */
+    FULLRBF,
```

Comment:
should this be `FULL_RBF`? (it may be better as-is but just thought I'd mention it.)

Reply:
Yes this would match the service bit naming.